securedrop/requirements/requirements.in

```diff
@@ -10,6 +10,7 @@ Flask-Babel>=1.0.0
 Flask-SQLAlchemy==2.5.1
 Flask-WTF>=1.0.0
 Flask>=2.0.3,<2.1.0
+flask-smorest<0.43.0  # last version to support Flask < 3.0.0
```

Comment:
I propose adding `flask-smorest` here to automate de/serialization (including validation) both here on the server side and potentially (via OpenAPI and TypeScript tooling) on the client side as well.  See `api2.py` for details, and of course this is subject to evaluation as a new set of dependencies.

Reply:
@legoktm will lead or delegate this evaluation next week.  Apart from reviewing the dependencies themselves, the options are (in my order of recommendation ;-):

1. Keep `flask-smorest` to generate an OpenAPI specification that the Client can consume, at least for JSON Schema validation and TypeScript typing, possibly up to and including code generation.
2. Remove `flask-smorest` (i.e., use ordinary Flask blueprints and routes) but keep Marshmallow (or, more natively, Python dataclasses + Pydantic) to generate *just* JSON Schemas that the client can consume for validation and typing.
3. Do manual validation here, as the v1 Journalist API does; and expect the Client to define its own schemas and types.